stratum/hal/lib/phal/db.proto

```diff
@@ -111,6 +112,17 @@ message ThermalGroup {
   repeated Thermal thermals = 1;
 }
 
+message OpticalTransceiver {
+  int32 id = 1;
+  string vendor_name = 2;
+  HwState hardware_state = 3;
+  uint64 frequency = 4;
+  float input_power = 5;
+  float output_power = 6;
+  float target_output_power = 7;
```

Comment:
I know that TAI only uses `float`, but could we, for consistency, use double here?
Also add unit: https://github.com/stratum/stratum/blob/7abcb6635cc30baa0b333f239dc59ac80044cd5d/stratum/hal/lib/phal/db.proto#L146-L148

Reply:
No name change yet.